app/src/core/design/components/userComponents/UtilComponents/VideoProperties.js

```diff
@@ -0,0 +1,225 @@
+            publicId = "";
+        }
+
+        setProp(props => {
+            props.props.thumbnailSrc = thumbnailSrcValue;
+            props.props.tempThumbnailSrc = tempThumbnailSrcValue;
+            props.props.publicId = publicId;
+            props.props.width = width;
+            props.props.height = height;
+        });
+    };
+
+    const handleClickThumbnail = async value => {
+        const formData = new FormData();
+        if (value != undefined) {
+            formData.append("file", value);
+
+            try {
+                const response = await exportImageUrl.generateUrl(formData);
+                width = response.width;
+                height = response.height;
+                thumbnailSrcValue = value;
+                tempThumbnailSrcValue = value;
+                publicId = response.public_id;
+            } catch (err) {
+                publicId = "";
+                tempThumbnailSrcValue = defaultThumbnailUrl;
+                thumbnailSrcValue = defaultThumbnailUrl;
+            }
+        } else {
+            thumbnailSrcValue = defaultThumbnailUrl;
+            publicId = "";
+            tempThumbnailSrcValue = defaultThumbnailUrl;
+        }
+        setProp(props => {
+            props.props.tempThumbnailSrc = tempThumbnailSrcValue;
+            props.props.thumbnailSrc = thumbnailSrcValue;
+            props.props.publicId = publicId;
+            props.props.width = width;
+            props.props.height = height;
+        });
+    };
+
+    return (
+        <CustomAccordion
+            title="Media"
+            preview={
+                <Tooltip title={props.props.src}>
+                    <IconButton size="small" href={props.props.src} target="_blank">
+                        <YouTubeIcon fontSize="small" htmlColor="#b4bec3" />
+                    </IconButton>
+                </Tooltip>
+            }
+            children={
+                <>
+                    <Box m={1}>
+                        <Typography variant="subtitle2" color="textSecondary">
+                            Media URL
+                        </Typography>
+                        <TextField
+                            variant="outlined"
+                            value={props.props.tempSrc === src ? "" : props.props.tempSrc}
+                            onChange={e => {
+                                e.persist();
+                                setProp(props => {
+                                    if (e.target.value !== "") props.props.tempSrc = e.target.value;
+                                    else props.props.tempSrc = src;
+                                });
+                            }}
+                            fullWidth
+                            margin="dense"
+                        />
+                        <ReplayOutlinedIcon
+                            onClick={() => handleClickReset(props.props.src)}
```

Comment:
rename to handleResetMediaURL, handleDoneMedialURL
handleResetThumbnailURL, handleDoneThumbnailURL